documentation/source/spelling_wordlist.txt

```diff
@@ -37,55 +41,90 @@ Wiki
 XML
 Xcelium
 backpressure
+bugfixes
+builtin
 callgraph
 cartesian
 cocotb
 coroutine
 coroutines
-dev
 diff
+disrecommended
```

Comment:
I've never seen this word before, and I don't think its in widespread use. "discouraged" is almost certainly more idiomatic.

Reply:
Just when I thought I could enrich the world with a new word!